### PR TITLE
Revert "Revert "Rich text support (#1077)""

### PR DESCRIPTION
**Pull request by im-shiv**
Reverts adobe/aem-core-forms-components#1129

### CODE REVIEW

File: bundles/af-core/src/test/resources/form/radiobutton/exporter-radiobutton-customized.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
   },
   "enforceEnum": true,
   "enumNames": [
-    "Item 1",
-    "Item 2"
+    "<p>Item 1</p>",
```
Comment (rismehta) [inline bundles/af-core/src/test/resources/form/radiobutton/exporter-radiobutton-customized.json:19]: Can you add a test case where if the options are not rich text, there is <p> tag ?

**Review by rismehta — CHANGES_REQUESTED** [review]
Please update the spec and fix the spec json present in the test collateral as part of this PR

File: ui.tests/test-module/specs/checkboxgroup/checkboxgroup.authoring.spec.js [hunk 2]
```diff
@@ -162,8 +171,71 @@ describe('Page - Authoring', function () {
...
+    it('check rich text inline editor is present', function(){
+        cy.openEditableToolbar(sitesSelectors.overlays.overlay.component + checkBoxGroupEditPathSelector);
+        cy.invokeEditableAction("[data-action='EDIT']");
+        cy.get(".rte-toolbar").should('be.visible');
+        cy.get('.rte-toolbar-item[title="Close"]').should('be.visible').click();
+    });
 
-  })
+    it('rule editor is working with rich text enum names', function(){
+        if(cy.af.isLatestAddon()){
```
Comment (rismehta) [inline ui.tests/test-module/specs/checkboxgroup/checkboxgroup.authoring.spec.js:208]: You don't need this check anymore. Please add test case for rule editor with/without rich text

Comment (im-shiv) [reply ui.tests/test-module/specs/checkboxgroup/checkboxgroup.authoring.spec.js:208]: yea, removed it

File: bundles/af-core/src/test/resources/form/radiobutton/exporter-radiobutton-customized.json (hunk 1 above)
Comment (im-shiv) [reply bundles/af-core/src/test/resources/form/radiobutton/exporter-radiobutton-customized.json:19]: aren't we considering by default that enumNames will be rich text?

**Pull request merged**